src/sail_json_backend/json.ml

```diff
@@ -91,60 +96,60 @@ let string_of_arg = function
 
 let rec string_list_of_mpat x = match x with
   | MP_aux (MP_lit ( l ), _) ->
-      prerr_endline ("MP_lit " ^ string_of_lit l);
+      if debug_enabled then prerr_endline ("MP_lit " ^ string_of_lit l);
```

Comment:
This logic is repeated with every instance. I suggest moving it into a new function, say `debug_print`?

Reply:
sure, will do.